test/testvm.py

```diff
@@ -1219,7 +1219,7 @@ def start_cockpit(self, atomic_wait_for_host="localhost"):
                 self.execute(script=RUN_COCKPIT_CONTAINER)
                 self.wait_for_cockpit_running(atomic_wait_for_host)
         else:
-            self.execute("systemctl start cockpit-testing.socket")
+            self.execute("mkdir -p /etc/systemd/system/cockpit.service.d/ && printf \"[Service]\nExecStartPre=-/bin/sh -c 'echo 0 > /proc/sys/kernel/yama/ptrace_scope'\nExecStart=\nExecStart=/usr/libexec/cockpit-ws --no-tls\n\" > /etc/systemd/system/cockpit.service.d/notls.conf && systemctl daemon-reload && systemctl start cockpit.socket")
```

Comment:
Can we make this a script (file) or a multiline string somehow? It's pretty long and will be difficult to read if a small part changes.